src/pages/home/home.ts

```diff
@@ -10,19 +10,23 @@ import { Geolocation } from '@ionic-native/geolocation';
 export class HomePage {
 
   // Insert default coordinates
-  latitude: number = 51.678418;
-  longitude: number = 7.809007;
+  latitude: number = 40.730610;
+  longitude: number = -73.935242;
 
   constructor(public navCtrl: NavController, private geolocation: Geolocation) {}
 
   getLocation(){
-  this.geolocation.getCurrentPosition().then((resp) => {
-    this.latitude = resp.coords.latitude;
-    this.longitude = resp.coords.longitude;
-  }).catch((error) => {
-    console.log('Error getting your location!', error);
-  });
+    this.geolocation.getCurrentPosition().then((resp) => {
+      this.latitude = resp.coords.latitude;
+      this.longitude = resp.coords.longitude;
+    }).catch((error) => {
+      console.log('Error getting your location!', error);
```

Comment:
@richardshu 

- [ ] Please write a better error logging statement here in case there is a reason the user has an error getting their current location, as the user will not be able to see the console.log. 

- [ ] Additionally, you want to mention in the message to said user, that the reason they might not be able to check their location is that they (the user) have blocked the app from getting geolocational information.

Reply:
Okay, I added the geolocation error message to pop up in an alert dialog box instead of showing up in the console.